src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -132,26 +104,22 @@ public void CanStartStakingViaAPI()
         [Fact]
         public void CanCallRPCMethodViaRPCsCallByNameAPI()
         {
-            try
+            using (NodeBuilder builder = NodeBuilder.Create())
             {
-                var fullNode = this.apiTestsFixture.stratisPowNode.FullNode;
-                var apiURI = fullNode.NodeService<ApiSettings>().ApiUri;
+                var posNode = builder.CreateStratisPosApiNode();
+                posNode.Start();
 
-                using (client = new HttpClient())
-                {
-                    client.DefaultRequestHeaders.Accept.Clear();
-                    client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                var apiURI = posNode.FullNode.NodeService<ApiSettings>().ApiUri;
 
-                    var response = client.GetStringAsync(apiURI + "api/rpc/callbyname?methodName=getblockhash&height=0").GetAwaiter().GetResult();
+                using (var httpClient = new HttpClient())
+                {
+                    httpClient.DefaultRequestHeaders.Accept.Clear();
+                    httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
 
-                    Assert.Equal("\"0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206\"", response);
+                    var response = httpClient.GetStringAsync(apiURI + "api/rpc/callbyname?methodName=getblockhash&height=0").GetAwaiter().GetResult();
```

Comment:
Bring the `apiURI` variable closer to where it's used 

![image](https://user-images.githubusercontent.com/16000683/38772081-5b17b3e8-4026-11e8-9fad-016d019410ba.png)

Reply:
Fixed